sw/device/mask_rom/mask_rom_start.S

```diff
@@ -114,12 +114,6 @@ _mask_rom_interrupt_vector:
   // is allocated executable space in ROM by the linker.
   .section .crt, "ax"
 
-  // Linker Relaxation is disabled until `__global_pointer$` is setup, below,
-  // because otherwise some sequences may be turned into gp-relative sequences,
-  // which is incorrect when `gp` is not initialized.
-  .option push
-  .option norelax
-
```

Comment:
Doesn't this mean that now we have linker relaxation enabled without GP being initialised?

Reply:
Yes. You're probably right that we should leave them disabled for safety. I'll fix it.

Reply:
Re-added the linker relaxations. Done.